packages/expo-modules-autolinking/README.md

```diff
@@ -11,8 +11,8 @@ Scripts that autolink Expo modules.
 
 # API documentation
 
-- [Documentation for the main branch](https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/module-autolinking.mdx)
 - [Documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/module-autolinking/)
+- [Documentation for the main branch](https://docs.expo.dev/versions/unversioned/sdk/module-autolinking/)
```

Comment:
These pages don't exist 😅 Maybe we can link to https://docs.expo.dev/modules/autolinking/ instead?

Reply:
Nice catch! Will updated those.